pypdf/_writer.py

```diff
@@ -829,18 +829,19 @@ def update_page_form_field_values(
                     if writer_annot.get(FieldDictionaryAttributes.FT) == "/Btn":
                         writer_annot.update(
                             {
-                                NameObject(
-                                    AnnotationDictionaryAttributes.AS
-                                ): NameObject(fields[field])
+                                NameObject(FieldDictionaryAttributes.V): NameObject(fields[field]),
+                                NameObject(FieldDictionaryAttributes.DV): NameObject(fields[field]),
```

Comment:
Note for myself:

> (Optional; inheritable) The default value to which the field reverts when
a reset-form action is executed (see 12.7.5.3, “Reset-Form Action”).
The format of this value is the same as that of V.